02 - Web UI Template/CognitiveSearch.UI/Search/DocumentSearchClient.cs

```diff
@@ -22,8 +22,8 @@ namespace CognitiveSearch.UI
     public class DocumentSearchClient
     {
         private IConfiguration _configuration { get; set; }
-        private SearchIndexClient _searchIndexClient;
-        private SearchClient _searchClient;
+        readonly SearchIndexClient _searchIndexClient;
```

Comment:
nit: include the access modifier:
`private readonly SearchIndexClient _searchIndexClient;`